core/dtn7/src/bin/dtnd.rs

```diff
@@ -93,6 +93,12 @@ async fn main() -> Result<(), std::io::Error> {
                 .help("Sets service discovery interval (0 = deactive, 2s = 2 seconds, 3m = 3 minutes, etc.) Refers to the discovery interval that is advertised when flag -b is set")
                 .takes_value(true),
         )
+        .arg(
+            Arg::new("disable_nd")
```

Comment:
- The initial problem was that the beacon interval in the ECLA is based on the ``-I`` setting as requested, so if you set it to 0 the beacon also won't work anymore. If you wanted to disable neighbour discovery for local testing but not the beacon sending there was no argument combination. 
- I still acknowledge that there might be a better way to handle that
   - Seperat intervals?

Reply:
or if you allow optional local configs similar to `-C` this could just be passed as another parameter to ecla. 
in the future I will probably add a nd selection mechanism (ipnd, mdns-sd, ecla, etc) which can also activate multiple discoveries and might also solve this problem.
so leaving it as is might be fine even though not very elegant.